tola_management/models.py

```diff
@@ -520,6 +520,13 @@ def diff_list_sorter(self, element):
         except ValueError:
             return 999
 
+    @property
+    def result_info(self):
```

Comment:
Can we add a comment to this to explain what's using it?  Just because indicators.models is a good demo of how we can load classes up with properties that we won't always remember to remove when we refactor later

Reply:
Good idea, done.